hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -43,89 +18,92 @@ This section describes all the configuration items available in the bootloader.
 
 The retry counters are reset when switching to the next boot mode.
 
-BOOT_ORDER fields  
+`BOOT_ORDER` fields  
 The BOOT_ORDER property defines the sequence for the different boot modes. It is read right to left and up to 8 digits may be defined.
 
-* 0x0 - NONE (stop with error pattern)
-* 0x1 - SD CARD
-* 0x2 - NETWORK
-* 0x3 - USB device boot - Reserved - Compute Module only.
-* 0x4 - USB mass storage boot (since 2020-09-03)
-* 0xf - RESTART (loop) - start again with the first boot order field. (since 2020-09-03)
+| Value | Mode      | Description                                                              |
+|-------|-----------|--------------------------------------------------------------------------|
+|  0x0  |  NONE     |  Stop and display error pattern                                          |
+|  0x1  |  SD CARD  |  SD card (or EMMC on Compute Module 4)                                   |
+|  0x2  |  NETWORK  |  Network boot                                                            |
+|  0x3  |  USB DEV  |  USB device boot - See [usbboot](https://github.com/raspberrypi/usbboot) |
+|  0x4  |  USB MSD  |  USB mass storage boot (since 2020-09-03)                                |
+|  0xf  |  RESTART  |  Start again with the first boot order field. (since 2020-09-03)         |
 
-Default: 0xf41 (0x1 in versions prior to 2020-09-03)  
+Default: `0xf41` (`0x1` in versions prior to 2020-09-03)  
 Version: 2020-04-16  
 
 * Boot mode `0x0` will retry the SD boot if the SD card detect pin indicates that the card has been inserted or replaced.
 * The default boot order is `0xf41` which means continuously try SD then USB mass storage.
+* USB device boot is intended for use with Compute Module 4 to load a debug image e.g. a Linux RAM disk. This should be the last boot option because it does not currently support timeouts or retries.
 
 ### MAX_RESTARTS
 If the RESTART (`0xf`) boot mode is encountered more than MAX_RESTARTS times then a watchdog reset is triggered. This isn't recommended for general use but may be useful for test or remote systems where a full reset is needed to resolve issues with hardware or network interfaces.
 
-Default: -1 (infinite)  
+Default: `-1` (infinite)  
 Version: 2020-09-03  
 
 ### SD_BOOT_MAX_RETRIES
 The number of times that SD boot will be retried after failure before moving to the next boot mode defined by `BOOT_ORDER`.  
 -1 means infinite retries.
 
-Default: 0  
+Default: `0`  
 Version: 2020-04-16  
 
 ### NET_BOOT_MAX_RETRIES
 The number of times that network boot will be retried after failure before moving to the next boot mode defined by `BOOT_ORDER`.  
 -1 means infinite retries.
 
-Default: 0  
+Default: `0`  
 Version: 2020-04-16  
 
 ### DHCP_TIMEOUT
 The timeout in milliseconds for the entire DHCP sequence before failing the current iteration.
 
-Minimum: 5000  
-Default: 45000  
+Minimum: `5000`  
+Default: `45000` 
```

Comment:
You've accidentally changed the two-spaces at the end of this line to a single-space, which means the forced line-break goes wrong.